forms-flow-web/src/_tests_/test-components/test-dashboard/dashboard.test.js

```diff
@@ -92,7 +92,6 @@ it("Should render the dashboard without breaking", () => {
     route: "/",
   });
   expect(screen.getByText("Metrics")).toBeInTheDocument();
-  expect(screen.getByText("Submissions")).toBeInTheDocument();
```

Comment:
Why this line removed

Reply:
This submission is no more appearing in the screen it is removes so it is not needed